src/store/domain/me/mutations.ts

```diff
@@ -28,6 +30,31 @@ export const mutations = defineMutations<S>()({
     if (!unreadChannel.channelId) throw 'addUnreadChannel: No Channel Id'
     Vue.set(state.unreadChannelsSet, unreadChannel.channelId, unreadChannel)
   },
+  upsertUnreadChannel(state: S, message: Message) {
+    const myId = state.detail?.id
+    const unreadChannel: UnreadChannel = {
+      channelId: message.channelId,
+      count: 1,
+      noticeable: false,
+      since: new Date().toISOString(),
+      updatedAt: new Date().toISOString()
+    }
+    const { detectMentions } = useDetecter(message.content)
+    unreadChannel.noticeable = detectMentions.some(data => data.id === myId)
+    if (
+      message.channelId in state.subscriptionMap ||
+      unreadChannel.noticeable
+    ) {
+      if (message.channelId in state.unreadChannelsSet) {
+        const preUnreadChannel = state.unreadChannelsSet[message.channelId]
+        unreadChannel.count = preUnreadChannel.count + 1
+        unreadChannel.since = preUnreadChannel.since
+        Vue.delete(state.unreadChannelsSet, unreadChannel.channelId)
+        Vue.set(state.unreadChannelsSet, unreadChannel.channelId, unreadChannel)
+      }
+      Vue.set(state.unreadChannelsSet, unreadChannel.channelId, unreadChannel)
+    }
+  },
```

Comment:
```suggestion
    const { detectMentions } = useDetecter(message.content)
    const noticeable = detectMentions.some(data => data.id === myId) //グループ宛てのメンションのチェックもしてほしい
    if (message.channelId in state.subscriptionMap || noticeable) {
      if (message.channelId in state.unreadChannelsSet) {
        const oldUnreadChannel = state.unreadChannelsSet[message.channelId]
        Vue.set(state.unreadChannelsSet, message.channelId, {
          ...oldUnreadChannel,
          count: oldUnreadChannel.count + 1,
          since: new Date().toISOString()
        })
      } else {
        Vue.set(state.unreadChannelsSet, unreadChannel.channelId, {
          channelId: message.channelId,
          count: 1,
          noticeable,
          since: new Date().toISOString(),
          updatedAt: new Date().toISOString()
        })
      }
    }
  },
```